utils/pstor/src/products/v1.rs

```diff
@@ -26,3 +28,16 @@ pub async fn detect_product_v1_prefix<S: Store>(store: &mut S) -> Result<bool, c
     let prefix = store.get_values_prefix(&key_prefix()).await?;
     Ok(!prefix.is_empty())
 }
+
+/// Deletes all v1 nexus_info by fetching all keys and parsing the key to UUID and deletes on
+/// success.
+pub async fn delete_all_v1_nexus_info<S: Store>(store: &mut S) -> Result<(), crate::Error> {
+    let kvs = store.get_values_prefix("").await.unwrap();
```

Comment:
return error

Reply:
Also we should probably use paging here, as I'm sure the clusters will be filled with these...

Reply:
Ahh, sorry missed this.

Reply:
I did try pagination, and it doesn't seem to work as expected. Would give another try.

Reply:
hmm maybe a bug on the paged method

Reply:
It worked. Have added the pagination.